modules/serverlesss-ossm-mtls-certs.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * serverless/service_mesh/serverless-ossm-without-kourier.adoc
+
+[id="serverlesss-ossm-mtls-certs_{context}"]
+= Creating a certificate to enable mTLS for external traffic
```

Comment:
This does not enable mTLS for external traffic. It enables TLS for it.

Reply:
Made some updates, lmk if this is better